_data/people.yml

```diff
@@ -134,6 +134,11 @@
    email: carson.kohlbrenner@colorado.edu
    pic: carson.jpg
 
+ - name: Caleb Kumar
+   role: Undergrad
```

Comment:
This should be lowercase

Reply:
We still have problems for some reason:

![image](https://github.com/HIRO-group/HIRO-group.github.io/assets/4378663/b6b7ca31-b264-43b7-97de-29d870e2d939)